TeamCode/src/main/res/xml/skystonev2.xml

```diff
@@ -28,19 +28,30 @@ SOFTWARE.
             <Motor name="leftRearDriveMotor"  port="2" /> <!-- green -->
             <Motor name="rightRearDriveMotor"  port="3" /> <!-- purple -->
 
+            <!-- Limit Switch -->
             <DigitalDevice name="deliveryMechLowLimitSwitch" port="1"/>
         </LynxModule>
 
         <LynxModule name="Secondary Hub" port="2">
             <LynxEmbeddedIMU name="imu 1" port="0" bus="0" />
 
+            <!-- DC Motors -->
             <Motor name="liftMotor" port="0" />
             <Motor name="leftIntakeMotor" port="1" />
             <Motor name="rightIntakeMotor" port="2" />
 
+            <!-- Servos -->
             <Servo name="fingerServo" port="0" />
             <Servo name="wristServo" port="1" />
             <Servo name="shoulderServo" port="3" />
+            <Servo name="leftGripServo" port="2" />
+            <Servo name="rightGripServo" port="4" />
```

Comment:
Is there some reason there's a gap between left and right, physically? I would expect them to be in 4 and 5.